_posts/blog/2017-05-31-conversational-interfaces.md

```diff
@@ -0,0 +1,54 @@
+# Creating Conversational Interfaces: Highlights from Google I/O 2017
+
+Natural language processing has exponentially improved over the last few years. As expected, one of the clear themes of Google I/O 2017 was building conversational interfaces.
+ 
+Having had the privilege of being on a team invited to create one of the first Actions on Google with transactional and multi-modal capabilities, I paid particular attention to sessions that focused on designing user experiences for voice interactions. While all the speakers had something unique to offer, I’d like to highlight a few repeating themes:
+
+## Make It A Conversation
+
+One of the unmistakable benefits of voice interactions is how efficiently they can understand and execute tasks; while a GUI might take several clicks and some typing, a voice command can be understood and executed in a matter of seconds. Leverage this advantage heavily, but use caution — the voice interface experience you design must be more than some way to bypass the typing and clicking involved in a traditional GUI. We tend to use the terms “voice” and “conversation” interchangeably (i.e. “voice user interface” equals “conversational user interface”). However, to truly reap the benefits of a voice interface, design your experience so the user feels like it is a conversation with another human.
+
+## Create Your Personality
+
+Give your product a real human voice.
+ 
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;“Exactly _why_ do I need to make the interaction sound like a personal conversation?” asks the analytical, dispassionate engineer. 
```

Comment:
I wonder if we can find a better solution for calling out lines than this indentation. As it is, it looks like a formatting error. Could this be a blockquote?

```markdown
> “Exactly _why_ do I need to make the interaction sound like a personal conversation?” asks the analytical, dispassionate engineer.
```